app/models/daos/JobDAO.scala

```diff
@@ -16,6 +17,28 @@ class JobDAO @Inject()(dbapi: DBApi)(implicit ec: DatabaseExecutionContext) {
 
   implicit private val parameters = Macro.toParameters[Job]
 
+  def find(id: Int): Future[Job] = Future {
```

Comment:
do you think we'd ever need to worry about passing in a job id that doesn't exist?

Reply:
Seems unlikely but I guess it's possible.